docs/tutorial/onchain-rln-relay-chat2.md

```diff
@@ -160,20 +160,20 @@ quitting...
 ## How to persist and reuse RLN credential
 
 You may pass the `rln-relay-cred-path` config option to specify a path for 1) persisting RLN credentials and 2) retrieving persisted RLN credentials.  
-RLN credential is persisted in the `rlnCredentials.txt` file under the specified path.
-If this file does not already exist under the supplied path, then a new credential is generated and persisted in the `rlnCredentials.txt` file.
+RLN credential is persisted in the `rlnKeystore.json` file under the specified path.
+If this file does not already exist under the supplied path, then a new credential is generated and persisted in the `rlnKeystore.json` file.
 Otherwise, the chat client does not generate a new credential and will use, instead, the persisted RLN credential.
 
 ```bash
-./build/chat2  --fleet:test --content-topic:/toy-chat/3/mingde/proto --rln-relay:true --rln-relay-dynamic:true --rln-relay-eth-contract-address:0x9C09146844C1326c2dBC41c451766C7138F88155  --rln-relay-eth-account-private-key:your_eth_private_key  --rln-relay-eth-client-address:your_sepolia_node  --ports-shift=1  --rln-relay-cred-path:./
+./build/chat2  --fleet:test --content-topic:/toy-chat/3/mingde/proto --rln-relay:true --rln-relay-dynamic:true --rln-relay-eth-contract-address:0x9C09146844C1326c2dBC41c451766C7138F88155  --rln-relay-eth-account-private-key:your_private_key  --rln-relay-eth-client-address:your_sepolia_node  --ports-shift=1  --rln-relay-cred-path:./rlnKeystore.json --rln-relay-cred-password:your_password
```

Comment:
I understood from the description above that the `--rln-relay-cred-path` is supposed to be a path to *folder* - since it explicitly states that is would persist the cred into `rlnKeystore.json` file under the given path, but the example specifies full path including the file 

So it seems either the doc or the example is not fully correct.

Reply:
Hmm, the given command works. Probably the doc doesn't explicitly mention that you have to pass a filename in. I'll make the change.

Reply:
Is this better? https://github.com/waku-org/nwaku/pull/1750/commits/f0c7fcb3c3fa64a548e79d3b002824d66af8bf57